app/src/main/java/org/coepi/android/MainActivity.kt

```diff
@@ -27,6 +30,9 @@ class MainActivity : AppCompatActivity() {
         onboardingShower.showIfNeeded()
 
         log.i("Testing the log")
+        
+        AppCenter.start(getApplication(), "0bb1bf95-3b14-48a6-a769-db1ff1df0307",
```

Comment:
This looks like a line of Java code, probably should be converted to Kotlin.
Ex. (https://github.com/microsoft/appcenter-sdk-android/issues/748)
`AppCenter.start(application, "0bb1bf95-3b14-48a6-a769-db1ff1df0307", Analytics::class.java, Crashes::class.java)`

Reply:
attempted to fix per suggestion. feel free to edit further if I still didn't get it right! thx @mellowgeek